### DISCUSSION TITLE
.turbo folder is not grading out

### DISCUSSION THREAD

**Discussion by Boby900** [vercel/turborepo#8640] [Help] [answered]
### Summary

.turbo folder is not grading out when I am adding the 
```bash

/+ .turbo/

.turbo

+ .turbo
```
I tried several ways and it's getting pushed to my remote repository.
Should it be pushed in the production code or not?

Thanks for your help in advance
![Screenshot (584)](https://github.com/vercel/turbo/assets/111265239/8b633bb5-5a8f-440b-85b5-a91e03329b11)

### Additional information

```js
It was pushed in the production, I had to delete that manually `.turbo` folder.
minimal repro = https://github.com/Boby900/yogurt
```

### Example

https://github.com/Boby900/yogurt

**Comment by Boby900** [comment]
never mind it's graded out after some time automatically but should it be pushed to production or not?

**Reply by anthonyshew** [reply, marked as answer]
It should be .gitignored as you're doing. The directory only contains information that is useful on your machine locally.

**Reply by Boby900** [reply]
@anthonyshew thanks for sharing this
